adserver/analyzer/tests.py

```diff
@@ -77,6 +78,10 @@ def test_get_analyzer_class(self):
             TextacyAnalyzerBackend,
         )
 
+    def test_normalize_title(self):
+        self.assertEqual(normalize_url("Title #"), "Title")
+        self.assertEqual(normalize_url("Deploy Your Own¶"), "Deploy Your Own")
```

Comment:
```suggestion
        self.assertEqual(normalize_title("Deploy Your Own¶"), "Deploy Your Own")
```

Shouldn't these be `normalize_title`?

Reply:
hah yea... copy/paste foul :(